### PR TITLE
chore: editorconfig + checker

### PR DESCRIPTION
**Pull request by mightyiam**

### CODE REVIEW

**Comment by mightyiam** [comment]
Based on #454.

File: tsconfig.configs.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "noEmit": true
+  },
+  "files": [
+    "./karma.conf.js",
+    "./gulpfile.js"
+  ]
+}
```
Comment (paldepind) [inline tsconfig.configs.json]: What is the purpose of this tsconfig file?

Comment (mightyiam) [reply tsconfig.configs.json]: Since this is from #454, I'll answer there.

Comment (mightyiam) [reply tsconfig.configs.json]: Does not exist any more.

**Pull request merged**